content/blog/bulk-onboarding-of-users-in-hpe-greenlake-edge-to-cloud-platform.md

```diff
@@ -0,0 +1,284 @@
+Enter HPE GreenLake Client Secret for Workspace HPEDEV -GLCP- Hackshack: ********************************
+Enter HPE GreenLake Client Secret for Workspace Super Awesome Company: ********************************                 
+Inviting user xxx@gmail.com to workspace  HPEDEV -GLCP- Hackshack                                              
+Inviting user yyy@lalli.fr to workspace  HPEDEV -GLCP- Hackshack                                                     
+Error sending invite for yyy@lalli.fr ! Already onboarded?
+Inviting user zzz@lalli.fr to workspace  Super Awesome Company
+Inviting user www@gmail.com to workspace  Super Awesome Company                                                
+Error sending invite for www@gmail.com ! Already onboarded?
+Done processing Excel file userlist.xlsx!
+
+1 user(s) invited to workspace HPEDEV -GLCP- Hackshack                                                                  
+List of users in workspace: HPEDEV -GLCP- Hackshack
+
+Username                     Status
+--------                     ------
+<email>                     VERIFIED
+…
+yyy@lalli.fr	            VERIFIED
+…
+xxx@gmail.com               UNVERIFIED
+…
+<email>                     VERIFIED
+
+1 user(s) invited to workspace Super Awesome Company                                                                    
+List of users in workspace: Super Awesome Company
+
+Username                          Status
+--------                          ------
+<email>                          VERIFIED
+…
+www@gmail.com 	                 VERIFIED
+…
+xxx@lalli.fr                     UNVERIFIED
+…
+<email>                          VERIFIED
+```
+
+As you can see, the script has invited 1 user in each workspace, the second email being already a member of the workspace (thus no action is necessary). 
+
+## What’s next?
+
+Through this post, I have shown you how it is possible to integrate with HPE GreenLake platform using the most popular scripting languages, such as PowerShell. You can get the source code for these scripts from [our community tooling repository](https://github.com/hpe-dev-incubator/GLP-API-Tooling).  
+
+If you’re interested in trying out what I just discussed, you might first want to check out one of our hands-on Workshops-on-Demand that lets you play with the HPE GreenLake APIs mentioned in this blog post. The workshops are free, available 24/7, and very easy to use. They give you a real-world experience without any risk. Check out our [catalog of workshops](https://developer.hpe.com/hackshack/workshops), register for the one you’re interested in and go! It’s as simple as that. 
+
+If you still have any questions regarding the HPE GreenLake platform APIs, join the [HPE Developer Community Slack Workspace](https://developer.hpe.com/slack-signup/) and start a discussion on our [\#hpe-greenlake-api](https://hpedev.slack.com/archives/C02EG5XFK8Q) channel. We are always here to help.
```

Comment:
Test suggestion to see if I can contribute to this repo (but the "any" isn't really needed).

```suggestion
If you still have questions regarding the HPE GreenLake platform APIs, join the [HPE Developer Community Slack Workspace](https://developer.hpe.com/slack-signup/) and start a discussion on our [\#hpe-greenlake-api](https://hpedev.slack.com/archives/C02EG5XFK8Q) channel. We are always here to help.
```